src/lib/createAuthContext.js

```diff
@@ -1,92 +1,195 @@
-import React, {createContext, useState, useEffect, useContext} from 'react'
-import authorize from './helpers/authorize'
-import { getCodeFromLocation } from './helpers/getCodeFromLocation'
-import { fetchToken } from './helpers/fetchToken'
-import { removeCodeFromLocation } from './helpers/removeCodeFromLocation'
-import { getVerifierFromStorage } from './helpers/getVerifierFromStorage'
-import { removeVerifierFromStorage } from './helpers/removeVerifierFromStorage'
-
-export default ({
-  clientId,
-  clientSecret,
-  provider,
-  scopes = [],
-  tokenEndpoint = `${provider}/token`,
-  storage = sessionStorage,
-  fetch = window.fetch,
-  busyIndicator = <>logging in...</>
-}) => {
-
-  const context = createContext({})
-  const {Provider} = context
-
-  class Authenticated extends React.Component {
-    static contextType = context
-    componentDidMount() {
-      const { ensureAuthenticated } = this.context
-      ensureAuthenticated()
+"use strict";
```

Comment:
It seems you transpiled some source files and committed the transpiled version. This is a no-no: In a pull request, you must change as little as possible to achieve the desired change (whilst you can always improve existing code while making change). You must leave the coding style chosen by previous developers intact. Committing a transpiled file as source file reduces readability, breaks the coding style of the original, etc.

You can re-do your changes based on the original files, adhering to this feedback, then request me to merge again.

Also, please note that the `npm run build` is supposed to be used to transpile the source into the `./dist` folder, check `package.json` for details.
 
For now I'll have to reject your pull request, sorry.

Reply:
I was using the internal files I had used to create my own solution, but it got mixed up with the original. I brought everything back to its original state.